try{
//     console.log("a+b=", a+c);

    
// }catch(err){
//     console.log(err);
    
// }